Time/rescuetime-activities.3m.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env PYTHONIOENCODING=UTF-8 python3
+# <xbar.title>RescueTime Activities</xbar.title>
+# <xbar.version>v1.4</xbar.version>
+# <xbar.author>Piotr Migdał</xbar.author>
+# <xbar.author.github>stared</xbar.author.github>
+# <xbar.desc>List your RescueTime activities in the status bar</xbar.desc>
+# <xbar.dependencies>python, Pillow (optional)</xbar.dependencies>
+# <xbar.image>https://github.com/stared/xbar-rescuetime-activities/blob/main/xbar-rescuetime-activities-screenshot.png</xbar.image>
+# <xbar.abouturl>https://github.com/stared/xbar-rescuetime-activities</xbar.abouturl>
+
+#
+# You need a RescueTime account and API key.
+# Generate the key at https://www.rescuetime.com/anapi/manage
+# Put the API key in ~/Library/RescueTime.com/api.key
+
+import os
+import json
+import datetime
+import urllib.parse
+import urllib.request
+
+
+API_KEY_PATH = os.path.expanduser("~/Library/RescueTime.com/api.key")
```

Comment:
Consider using [xbar variables](https://xbarapp.com/docs/2021/03/14/variables-in-xbar.html) to manage this. Would make download and setup easier for others.

Reply:
Thank you - I modified that.